packages/scene-composer/src/components/panels/TopBar.tsx

```diff
@@ -59,29 +62,35 @@ export const TopBar: FC = () => {
     [setActiveCameraSettings],
   );
 
-  return (
-    <StyledSpaceBetween direction='horizontal' size='xxs'>
-      <ButtonDropdown
-        data-testid={'view-options'}
-        items={[
-          {
-            id: KnownComponentType.MotionIndicator,
-            text: intl.formatMessage({
-              defaultMessage: 'Motion indicator',
-              description: 'dropdown button option text for motion indicator component',
-            }),
-            iconSvg: motionIndicatorVisible ? <Checked /> : <></>,
-          },
-        ]}
-        onItemClick={settingsOnItemClick}
-      >
-        {intl.formatMessage({ defaultMessage: 'View Options', description: 'view options dropdown button text' })}
-      </ButtonDropdown>
-      {cameraViewEnabled && (
-        <ButtonDropdown data-testid={'camera-views'} items={cameraItems} onItemClick={setActiveCameraOnItemClick}>
-          {intl.formatMessage({ defaultMessage: 'Cameras', description: 'camera views dropdown button text' })}
-        </ButtonDropdown>
-      )}
-    </StyledSpaceBetween>
-  );
+  if (showTopBar) {
+    return (
```

Comment:
You could write this as ```{showTopBar && (<JSX_Content />)}``` & it will only populate if `showTopBar` is true. That way you can remove the need to return an empty fragment (`<></>`).